RELEASE.md

```diff
@@ -0,0 +1,10 @@
+## Procedures
+* make a PR to update:
+  * kube-state-metrics image tag for both `quay.io` and `k8s.gcr.io`
+  * compatibility matrix
+  * change log
+  * kube-state-metrics image tag used in Kubernetes deployment yaml config
+* cut the new release branch, i.e., `release-1.2`
```

Comment:
or merge/cherry-pick changes onto the minor release branch you intend to tag the release on

Reply:
Will do.